analyses/infercnv-consensus-cell-type/references/diagnosis-celltype-groups.tsv

```diff
@@ -0,0 +1,15 @@
+diagnosis_group	celltype_groups
+B-cell leukemia	dendritic cell, innate lymphoid cell, macrophage, monocyte, myeloid, natural killer cell, plasma cell, T cell, endothelial cell, pericyte
+T-cell leukemia	B cell, dendritic cell, innate lymphoid cell, macrophage, monocyte, myeloid, natural killer cell, plasma cell, endothelial cell, pericyte
+Myeloid leukemia	B cell, dendritic cell, innate lymphoid cell, macrophage, natural killer cell, plasma cell, T cell
+Mixed-type leukemia	dendritic cell, innate lymphoid cell, macrophage, natural killer cell, plasma cell, endothelial cell, pericyte
+Sarcoma	B cell, dendritic cell, innate lymphoid cell, macrophage, monocyte, myeloid, natural killer cell, plasma cell, T cell, endothelial cell, pericyte

```

Comment:
Bone sarcomas have similar phenotypes to monocytes, so I would remove those from the list. I think keeping myleoid is okay, but monocytes in particular I would remove.